fidesctl/Dockerfile

```diff
@@ -61,3 +64,26 @@ ENV RUNNING_IN_DOCKER=TRUE
 
 EXPOSE 8080
 CMD ["fidesctl", "webserver"]
+
+############################
+## Application Dev Setup ##
+###########################
+FROM builder as dev
+
+# Install fidesctl as a symlink
+RUN pip install -e ".[all,mssql]"
+
+
+#############################
+## Application Prod Setup ##
+############################
+FROM builder as prod
+
+# Remove anything that could influence the build
+RUN rm -rf src/fidesctl.egg-info/
```

Comment:
Could the `src/fidesctl.egg-info/`, `build/`, and `dist/` be added to the `.dockerignore` file and have the same result? If so that is a couple less steps for the build.

Reply:
they definitely could, i just put it here to be super sure

I'll move them to the ignore file and make sure it still works as intended